### DISCUSSION TITLE
Permissions for the player

### DISCUSSION THREAD

**Discussion by Aisponch** [TownyAdvanced/Towny#5915] [Q&A]
There is a command /town and an abbreviation /t if you issue an op, then the command /t will work correctly, but only the command /towny without abbreviations is available to an ordinary player. How can I issue a permit?

**Comment by LlmDl** [comment]
Another plugin is likely stealing the `/t` alias. Use the server commands.yml to specify which plugin uses `/t`.

**Reply by Aisponch** [reply]
the /t commands work but are highlighted in red

**Reply by LlmDl** [reply]
That's not something I can do much about I think.

**Comment by ggvbo** [comment]
Did you solve this?
